spec/rails/date_spec.rb

```diff
@@ -13,4 +13,8 @@
     expect(converted.day).to eq date.day
     expect(converted.time_zone).to eq Time.zone
   end
+
+  it "should encode with the configured timezone" do
+    expect(date.at_time(12, 0, 0).zone).to eq Time.zone.now.zone
+  end
```

Comment:
Can we please add a test where there's only 1 argument (e.g. a time argument) too.

Thanks